### PR TITLE
[3573][15.0][MIG] purchase_order_cancel: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3573](https://www.quartile.co/web#id=3573&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: purchase_order_cancel/static/description/index.html
Comment (yostashiro) [inline purchase_order_cancel/static/description/index.html:1]: Shall we remove this file and images, as it adds a bit to the maintenance cost?

**Review by yostashiro — COMMENTED** [review]
Please regenerate the ja.po file.

File: purchase_order_cancel/i18n/ja.po [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: purchase_order_cancel
+#: model_terms:ir.ui.view,arch_db:purchase_order_cancel.purchase_order_cancel_form_view
+msgid "Are you sure to cancel the selected purchase order(s)?"
+msgstr "選択した注文書をキャンセルしてもよろしいですか?"
```
Comment (kanda999) [inline purchase_order_cancel/i18n/ja.po]:
```suggestion
msgstr "選択した発注を取り消しますか?"
```

File: purchase_order_cancel/i18n/ja.po [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+#. module: purchase_order_cancel
+#: model_terms:ir.ui.view,arch_db:purchase_order_cancel.purchase_order_cancel_form_view
+msgid "Cancel"
+msgstr "取消"
+
+#. module: purchase_order_cancel
+#: model_terms:ir.ui.view,arch_db:purchase_order_cancel.purchase_order_cancel_form_view
+msgid "Cancel Purchase Order"
+msgstr "注文書のキャンセル"
```
Comment (kanda999) [inline purchase_order_cancel/i18n/ja.po]:
```suggestion
msgstr "発注取消"
```

File: purchase_order_cancel/i18n/ja.po [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+#. module: purchase_order_cancel
+#: model_terms:ir.ui.view,arch_db:purchase_order_cancel.purchase_order_cancel_form_view
+msgid "Confirm"
+msgstr "確認"
+
+#. module: purchase_order_cancel
+#: model:ir.model.fields,field_description:purchase_order_cancel.field_purchase_order_cancel__create_uid
+msgid "Created by"
+msgstr ""
```
Comment (kanda999) [inline purchase_order_cancel/i18n/ja.po]:
```suggestion
msgstr "作成者"
```

**Review by kanda999 — APPROVED** [review]
Function review.

Please adjust translation.

**Comment by yostashiro** [comment]
@kanda999 Please merge this and update the module list.

**Pull request merged**